TINY-12194: Changelogs (#10763)
tinymce/tinymce: .changes/tinymce/8.3.0.md

## 8.3.0 - 2025-12-10

### Added
- Can now remove a command using the `editorCommands` API. #TINY-13144
- New options `files_upload_handler`, `documents_file_types`, and `link_uploadtab`. #TINY-13278
- A new upload tab to the link dialog. #TINY-13278
- New options for `dropzone` dialog components: `dropAreaLabel`, `buttonLabel`, `allowedFileTypes` and `allowedFileExtensions`. #TINY-13278

### Improved

> - Accordions can now be opened and closed when the editor is readonly. #TINY-12316

Reviewer: ```suggestion
- Accordions can now be opened and closed when the editor is in a read-only state. #TINY-12316
```

- Show editor notification when a premium plugin is not allowed for a given license key. #TINY-12937

### Fixed
- Help Plugin did not contain an entry for the Fullpage HTML plugin. #TINY-13183

> - Accordions could be toggled when the editor was disabled. #TINY-12315

Reviewer: ```suggestion
- Accordions could be toggled when the editor was in a disabled state. #TINY-12315
```

- Removing a line height sometimes did not remove it from all selected lines. #TINY-13053
- The `font-size` style on `li` elements was not copied to new list items. #TINY-13224

> - Toolbar split button chevron tooltip is now the same as the main button tooltip unless explicitly set otherwise. #TINY-13271

Reviewer 2: This sounds odd?
```suggestion
- Toolbar split button chevron tooltip is now the same as the main button tooltip unless explicitly set. #TINY-13271
```
